cpp/inline/mcts/SearchLog.inl

```diff
@@ -0,0 +1,97 @@
+#include <mcts/SearchLog.hpp>
+
+namespace mcts {
+
+template <core::concepts::Game Game>
+void SearchLog<Game>::build_graph(Graph<Game>& graph) {
+  using State = Game::State;
+  using edge_t = typename Node<Game>::edge_t;
```

Comment:
With `using` statements, you don't need `typename` on the right-hand-side. I prefer to remove it for cosmetics.

Reply:
fixed